easy-ui-react/src/Banner/Banner.module.scss

```diff
@@ -0,0 +1,77 @@
+@use "../styles/common" as *;
+
+.variantSuccess {
+  @include component-token(
+    "banner",
+    "color",
+    theme-token("color.text.success.inverse")
+  );
+  @include component-token(
+    "banner",
+    "background.color",
```

Comment:
**nit:** in my previous review comment i was thinking this could be `color.background` to stick with our paradigm of `<foundation>.<property>.<modifiers>`